docker-compose.yml

```diff
@@ -1,12 +1,22 @@
 version: "3.9"
 
 services:
-  dynamodb:
-    image: amazon/dynamodb-local:latest
+  localstack:
+    image: localstack/localstack:latest
```

Comment:
Because Localstack includes DynamoDB, I removed the `dynamodb` container from the docker compose file.